src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -473,6 +486,87 @@ public Void call() throws Exception {
             }
             return null;
         }
+        
```

Comment:
@dean-e-clark there should be no spaces in empty lines, IntelliJ IDEA should remove them by default